src/components/Meta.tsx

```diff
@@ -37,29 +44,20 @@ export default function render(
           "docs sectionType-" + props.sectionType + " " + props.category
         }
       />
+      {/* SEO content */}
+      <meta name="description" content={props.fields?.seoDescription || props?.highlights}/>
+        
+      <meta property="og:image" content={ogImageURL || ogImage} />
+      <meta name="twitter:image" content={ogImageURL || ogImage} />
+      
+      <meta name="robots" content={props.fields?.robotsTag || props?.robots} />
```

Comment:
Noting that the contentful default here is "FOLLOW", lets make sure that is handled appropriately

Reply:
Also need to make sure this is handled well for the content within the code that does not have any robots information